test/framework/kogitoserviceutils.go

```diff
@@ -57,6 +59,24 @@ func installOrDeployService(serviceHolder *bddtypes.KogitoServiceHolder, install
 	return err
 }
 
+// enableUseKogitoInfraIfInfinispanURLsNotDefined sets Kogito service to use KogitoInfra in case persistence is enabled and Infinispan URL is not defined
+func enableUseKogitoInfraIfInfinispanURLsNotDefined(serviceHolder *bddtypes.KogitoServiceHolder) {
+	if infinispanAware, ok := serviceHolder.GetSpec().(v1alpha1.InfinispanAware); ok {
+		if serviceHolder.EnablePersistence && len(infinispanAware.GetInfinispanProperties().URI) == 0 {
+			infinispanAware.GetInfinispanProperties().UseKogitoInfra = true
```

Comment:
So If I specify both `EnablePersistence` and URI for Infinispan, what will happen? Will operator ignore it? Was this change done only to have abstraction and not rely on  `UseKogitoInfra` flag?

Reply:
If you specify `EnablePersistence` and Infinispan URI then `UseKogitoInfra` will stay false and application will connect to external Infinispan.
The change was done to simplify configuration tables for applications, in previous approach both `EnablePersistence` and `UseKogitoInfra` were specified in config table. That approach was unnecessary verbose as it is possible to derive `UseKogitoInfra` value based on `EnablePersistence` and availability of Infinispan URL.